python/mcap/tests/test_writer.py

```diff
@@ -115,3 +117,34 @@ def test_out_of_order_messages():
     chunk_index = next(r for r in records if isinstance(r, ChunkIndex))
     assert chunk_index.message_start_time == 0
     assert chunk_index.message_end_time == 100
+
+@pytest.mark.parametrize(
+    "null_schema", [(True,), (False,)]
+)
+def test_generate_sample_cbor_data(null_schema: bool):
```

Comment:
This is the wrong place for this test, but we should test this functionality somewhere since it was also the source of a SIGSEV...

Reply:
Where should we put this test?